webonary-cloud-api/lambda/package.json

```diff
@@ -1,14 +1,12 @@
 {
   "name": "lambda",
   "version": "0.1.0",
-  "devDependencies": {
-    "@types/mongodb": "^3.5.20"
-  },
+  "devDependencies": {},
   "dependencies": {
     "aws-lambda": "^1.0.6",
     "aws-sdk": "^2.691.0",
     "axios": "^0.27.2",
-    "mongodb": "^3.5.8",
+    "mongodb": "^4.7.0",
```

Comment:
The version in Lambda was older than the main project, which was causing build and test from the main project to fail sometimes. Now both versions are the same.